crates/binstalk/src/lib.rs

```diff
@@ -4,7 +4,7 @@ pub mod errors;
 pub mod fetchers;
 pub mod fs;
 pub mod helpers;
-pub mod manifests;
+pub use binstalk_manifests as manifests;
```

Comment:
that should be move up to the `use`s

Reply:
I actually moved it down since the `use` is at the bottom.